web/tests/test_documenten.py

```diff
@@ -0,0 +1,71 @@
+import os.path
+from unittest import TestCase
+
+# Prepare environment
+from flask_testing import TestCase as FlaskTestCase
+from mock import patch
+
+os.environ['FOCUS_USERNAME'] = 'FOCUS_USERNAME'
+os.environ['FOCUS_PASSWORD'] = 'FOCUS_PASSWORD'
+os.environ['FOCUS_WSDL'] = 'focus/focus.wsdl'
+os.environ['TMA_CERTIFICATE'] = __file__
+
+from focus.config import PROJECT_PATH, config, credentials  # noqa: E402  Module level import not at top of fil
+from focus.focusconnect import FocusConnection  # noqa: E402  Module level import not at top of fil
```

Comment:
Duplicate noqa uitleg ("Module level import not at top of fil") kan weg, of bij allemaal toevoegen. Nu hebben twee van de drie imports deze uitleg.